tfe/resource_tfe_policy_set.go

```diff
@@ -115,7 +115,12 @@ func resourceTFEPolicySet() *schema.Resource {
 
 						"oauth_token_id": {
 							Type:     schema.TypeString,
-							Required: true,
+							Optional: true,
```

Comment:
Since `"oauth_token_id"` and `"github_app_installation_id"` are both optional but at least one must be specified, we should add the `AtLeastOneOf` and `ConflictWith` validations, what do you think?